example6/fitnesse/fitnesse.py

```
@@ -0,0 +1,32 @@
+class Fitnesse:
+    @staticmethod
+    def testable_html(page_data, include_suite_setup):
+        wiki_page = page_data.get_wiki_page()
+        buffer = []
+        if page_data.has_attribute("Test"):
+            if include_suite_setup:
+                suite_setup = PageCrawlerImpl.get_inherited_page(SuiteResponder.SUITE_SETUP_NAME, wiki_page)
```

Ruff (0.8.2)

Undefined name `PageCrawlerImpl`

(F821)